docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md

```diff
@@ -117,32 +115,12 @@ Both the `kotlinx.serialization` and SQLDelight libraries also require additiona
    ```kotlin
        plugins {
        // ...
-       kotlin("plugin.serialization") version "%kotlinVersion%"
-       id("com.squareup.sqldelight")
+       kotlin("plugin.serialization").version("%kotlinVersion%")
+       id("com.squareup.sqldelight").version("%sqlDelightVersion%")
    }
    ```
 
-3. Now go to the `build.gradle.kts` file in the project _root directory_ and specify the classpath for the plugin in the
-   build system dependencies:
-
-    ```kotlin
-    buildscript {
-        
-        dependencies {
-            // ...
-            classpath("com.squareup.sqldelight:gradle-plugin:%sqlDelightVersion%")
```

Comment:
Is there any reason to specify buildscript dependencies explicitly? I think the explicit version [in the plugins](https://github.com/JetBrains/kotlin-web-site/pull/3824/files#diff-6d95cf38840a09a14676b740a25fa2dde290007c571510ff620ef14bb077cb9aR119) block achieves the same goal